pkg/asset/installconfig/powervs/client.go

```diff
@@ -746,3 +750,30 @@ func (c *Client) ServiceInstanceIDToCRN(ctx context.Context, id string) (string,
 
 	return "", nil
 }
+
+// GetDatacenterCapabilities retrieves the capabilities of the specified datacenter.
+func (c *Client) GetDatacenterCapabilities(ctx context.Context, region string) (map[string]bool, error) {
```

Comment:
Can we wrap or format the errors here to include the original error ?

`fmt.Errorf("...: %w", err)`

Reply:
Definitely, good catch! Reflected in the latest force-push.